main.js

```diff
@@ -1,32 +1,29 @@
-        let btn = document.querySelector("button");
-        btn.addEventListener("click", countDown);
-               
-        let hours = "0";
-        let minutes = "0";
-        let seconds =  "10";
+let btn = document.querySelector("button");
+btn.addEventListener("click", countDown);
 
-        hours < 10 ? hours = '0' + hours : hours = hours;
-        minutes < 10 ? minutes = '0' + minutes : minutes = minutes;
-        seconds < 10 ? seconds = '0' + seconds : seconds = seconds;
-        
-        let countContainer = document.getElementById('count');
+let hours = "0";
+let minutes = "0";
+let seconds =  "10";
 
-        
-        function countDown(){       
-            
-             setInterval(() => {
-                let watch = hours + ':' + minutes + ':' + seconds;
-                seconds!="00" ?
-                seconds-- :
-                seconds = '00';
-                countContainer.innerHTML = watch;
-                clearInterval();
-             }, 1000);
+hours < 10 ? hours = '0' + hours : hours = hours;
+minutes < 10 ? minutes = '0' + minutes : minutes = minutes;
+seconds < 10 ? seconds = '0' + seconds : seconds = seconds;
 
+let countContainer = document.getElementById('count');
 
-             function stopInterval(){
-                window.clearInterval();
-            }
-                    
-            }
-            
\ No newline at end of file
+function countDown() {
+   let thisInterval = setInterval(() => {
+   let watch = hours + ':' + minutes + ':' + seconds;
+   countContainer.innerHTML = watch;
+   if(Number(seconds) === 0) {
```

Comment:
I don't understand this, please

Reply:
I'm making sure that seconds is a number